scriptshifter/tables/data/abkhaz_cyrillic.yml

```diff
@@ -0,0 +1,81 @@
+general:
+  name: Abkhaz (Cyrillic)

```

Comment:
This needs an entry in `index.yml` to be usable. You can use the other entries as example. It's best to add entries alphabetically by label, so this would be the first entry.

Reply:
Thank you, I clearly should have re-read the configuration documentation before trying to add a new file. It's right there at the end.

Reply:
Thank you! Clearly I should have re-read your documentation before doing something new, the information on the index file is right at the end of the configuration information.